### ISSUE TITLE
Bug: Not showing invidious redirect instances

### ISSUE THREAD

**Issue by kogeletey** [iv-org/invidious-redirect#33]
The following problem occured in the firefox today
![8439](https://github.com/iv-org/invidious-redirect/assets/39346864/0257531d-6f82-4762-8c72-13158f9c1e4d)
And the loading is never ending

 The following error appears in the browser console:
``` entry[1].monitor.dailyRatios is undefined```

 Thanks for your product, hope you fix it soon

**Comment by TheFrenchGhosty** [comment]
It has been discussed on Matrix yesterday, the problem is that uptimerobot stopped providing "dailyRatios" in their response.

I just made a hotfix. In short: I made it so the dailyRatios isn't looked in the uptimerobot response anymore... Instead it just display "100". It's dirty, but it will work until uptimerobot fixes it or until it gets properly reworked. https://github.com/iv-org/invidious-redirect/pull/34
